src/Yoti/YotiClient.php

```diff
@@ -131,10 +132,9 @@ public function getOutcome()
      *
      * @param string $encryptedConnectToken
      *
-     * @return \Yoti\ActivityDetails
+     * @return ActivityDetails
      *
-     * @throws \Exception
-     * @throws \Yoti\Exception\ActivityDetailsException
+     * @throws ActivityDetailsException
```

Comment:
So, should this be here?  Or should it be removed?

Reply:
If you are referring to `ActivityDetailsException` yes it should stay there.